scripts/generate_v1.1_data.py

```diff
@@ -0,0 +1,90 @@
+"""Generates v1.1 Data.
+
+From v1.0 data, we include additional columns containing parsed
+attribute values for the superset of linguistic attributes across all
+languages.
+"""
+
+import csv
+from copy import copy
+from typing import Dict, List
+
+# Path to the corpus.
+CORPUS_PATH = "data/clse_v1.0.csv"
+NEW_CORPUS_PATH = "data/clse_v1.1.csv"
+
+# List of linguistic signature prefixes.
+VERBOSE_SIGNATURE_PREFIXES = [
+    "InflectedNounForm.",
+    "NominalInflectedForm.",
+    "NameTags.",
+    "Variant.",
+    "WordStemAnnotation.",
+    "Common",
+    "SurfaceForm.",
+    "Determiner.",
+    "Proper",
+    "InflectedForm.",
+]
+
+
+def get_linguistic_attribute_dict(linguistic_signature: str) -> Dict[str, str]:
+    """Returns a dictionary of linguistic attribute -> value."""
+    # Strip verbose prefixes from the linguistic signature.
+    reduced_linguistic_signature = linguistic_signature
+    for verbose_signature_prefix in VERBOSE_SIGNATURE_PREFIXES:
+        reduced_linguistic_signature = reduced_linguistic_signature.replace(
```

Comment:
here you are not checking that those are really prefixes. I suggest you strip the prefixes down below instead